engine/runtime/src/main/java/org/enso/compiler/Cache.java

```diff
@@ -2,7 +2,7 @@
 
 import com.oracle.truffle.api.TruffleFile;
 import com.oracle.truffle.api.TruffleLogger;
-import org.bouncycastle.jcajce.provider.digest.SHA3;
+import org.bouncycastle.jcajce.provider.digest.SHA1;
```

Comment:
Why do we use bouncycastle and not the standard JVM implementation of the digest?

Reply:
Looks like the only other place where we use SHA3 is for Buffers. Those could also be downgraded, I believe.